compiler/bindings/python/CMakeLists.txt

```diff
@@ -14,6 +14,12 @@
 # understand what this does.
 ################################################################################
 
+# This relies on bundled-LLVM, indicated by IREE_LLD_TARGET, as opposed to
+# installed-LLVM (bring-your-own-LLVM).
+if(NOT IREE_LLD_TARGET)
+  return()
+endif()
```

Comment:
Ah, I reviewed files in a strange order. This feels weird to me - why can't we build the compiler's python bindings with an external LLVM? Is the external LLVM binaries-only, or is there source code? If _that's_ why, then we should make these checks state that more clearly.

Reply:
Yes, the opposite of `bundled-llvm` is `installed-llvm` and that means breaking the source dependency.